ably/realtime/realtime_channel.py

```diff
@@ -40,17 +41,17 @@ class RealtimeChannel(EventEmitter, Channel):
         Unsubscribe to messages from a channel
     """
 
-    def __init__(self, realtime, name):
+    def __init__(self, realtime, name: str):
```

Comment:
we should add a typing for the realtime param here

Reply:
That causes a circular import

Reply:
you can work around that by doing this:

```python
from __future__ import annotations
from typing import TYPE_CHECKING

if TYPE_CHECKING:
    from ably.realtime.realtime import AblyRealtime
```